### ISSUE TITLE
T4.x reaches 2542.75 with GCC 10.3.1,  -O2  -funroll-all-loops

### ISSUE THREAD

**Issue by FrankBoesing** [PaulStoffregen/CoreMark#14]
`2K performance run parameters for coremark.`  
`CoreMark Size    : 666`  
`Total ticks      : 15731`  
`Total time (secs): 15.73`  
`Iterations/Sec   : 2542.75`  
`Iterations       : 40000`  
`Compiler version : GCC10.3.1 20210621 (release)`  
`Compiler flags   : (flags unknown)`  
`Memory location  : STACK`  
`seedcrc          : 0xE9F5`  
`[0]crclist       : 0xE714`  
`[0]crcmatrix     : 0x1FD7`  
`[0]crcstate      : 0x8E3A`  
`[0]crcfinal      : 0x25B5`  
`Correct operation validated. See README.md for run and reporting rules.`  
`CoreMark 1.0 : 2542.75 / GCC10.3.1 20210621 (release) (flags unknown) / STACK`

**Comment by FrankBoesing** [comment]
Still slower than IAR with STM: https://www.eembc.org/viewer/?benchmark_seq=2533

**Comment by fab672000** [comment]
> Still slower than IAR with STM: https://www.eembc.org/viewer/?benchmark_seq=2533

Wrong: CoreMark on this page is 1082.4, which is good but not quite close to the T4.
Now if you look at the latest H7's from ST it gets more interesting and better than T4's but not by much:
https://www.eembc.org/viewer/?benchmark_seq=13276
